ipn/ipnlocal/web_client.go

```diff
@@ -34,57 +34,56 @@ type webServer struct {
 func (b *LocalBackend) SetWebLocalClient(lc *tailscale.LocalClient) {
```

Comment:
`SetWebClientLocalClient` is really tough on the tongue but is technically more accurate...

Reply:
Could actually be `SetLocalClient` with the idea that there may eventually be other consumers of `lc` from ipnlocal (although honestly not sure that will ever be true).

Reply:
Will keep as-is, can be renamed in a future change if desired.